attributes/default.rb

```diff
@@ -143,6 +143,9 @@
 
 # Configure Abiquo websockify
 default['abiquo']['websockify']['port'] = 41337
+default['abiquo']['websockify']['api_url'] = 'https://localhost/api'
```

Comment:
Does it make sense to infer this [the same way we do in the UI recipe](https://github.com/abiquo/abiquo-cookbook/blob/master/recipes/setup_ui.rb#L18-L23)?